mod/scripts/vscripts/sh_AutoKick.gnut

```diff
@@ -1,6 +1,6 @@
 global function Autokick_Init
 void function Autokick_Init() {
-	AddPrivateMatchModeSettingEnum("#MODE_SETTING_CATEGORY_SERVERTOOLS", "AutoKick", ["#SETTING_DISABLED", "#SETTING_ENABLED"], "0")
+	//AddPrivateMatchModeSettingEnum("#MODE_SETTING_CATEGORY_SERVERTOOLS", "AutoKick", ["#SETTING_DISABLED", "#SETTING_ENABLED"], "0")
```

Comment:
I'm not sure what to do with this, since AutoKick can now be any integer, not just 0 or 1

Reply:
its unlikely people will be setting autokick rules from private match lobby anyway, but we can just use AddPrivateMatchModeSettingArbitrary to allow numerical values instead of enabled/disabled